cfme/tests/infrastructure/test_host.py

```diff
@@ -3,12 +3,14 @@
 import socket
 
 import pytest
+from selenium.common.exceptions import UnexpectedAlertPresentException
```

Comment:
Its better to import this from `widgetastic.exceptions`, because if we wrap it or change webdrivers then that change will propagate to here without having to modify.

Reply:
I don't see this change? Was it pushed?